cql3/expr/expression.hh

```diff
@@ -281,6 +281,8 @@ extern bool is_on_collection(const binary_operator&);
 /// column_value.
 extern expression replace_column_def(const expression&, const column_definition*);
 
+extern expression replace_token(const expression&, const column_definition*);
+
```

Comment:
When we collect more of these, we can consolidate with a generic replace(expression, Predicate, Transform)